src/components/AddComment.jsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect } from "react";
+import { useState } from "react";
+import "../App.css";
+import { postComment } from "../api";
+
+function AddComment({ setComments, review_id }) {
+  const [newComment, setNewComment] = useState("");
+
+  const handleChange = (event) => {
+    setNewComment(event.target.value);
+  };
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    postComment(review_id, newComment)
+      .then((commentsApi) => {
+        setNewComment("");
+        setComments((currComments) => {
+          const newComments = [...currComments];
+          newComments.unshift(commentsApi);
+          return newComments;
+        });
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
```

Comment:
We want to make sure that users aren't able to add blank comments - we will need to add some extra logic to prevent this! 🙂